src/vrm/VRMUtils/extractThumbnailBlob.ts

```diff
@@ -25,13 +25,15 @@ export function extractThumbnailBlob(renderer: THREE.WebGLRenderer, vrm: VRM, si
     throw new Error('extractThumbnailBlob: This VRM does not have a thumbnail');
   }
 
+  const canvas = renderer.getContext().canvas;
+
   // store the current resolution
   renderer.getSize(_v2A);
   const prevWidth = _v2A.x;
   const prevHeight = _v2A.y;
 
   // overwrite the resolution
-  renderer.setSize(size, size);
+  renderer.setSize(size, size, !(canvas instanceof OffscreenCanvas));
```

Comment:
Oh! I didn't know the third argument of setSize is `true` by default.

There is no reason resizing the style in this function, could you set false here instead? I also want to change `revert to previous resolution` one of course.

Reply:
Do you mean all `renderer.setSize()` in this method do not need to updateStyle?

Reply:
yes